src/ccache.cpp

```diff
@@ -159,30 +159,23 @@ static pid_t compiler_pid = 0;
 static const char HASH_PREFIX[] = "3";
 
 static void
-add_prefix(const Context& ctx, Args& args, const char* prefix_command)
+add_prefix(const Context& ctx, Args& args, const std::string& prefix_command)
 {
-  if (str_eq(prefix_command, "")) {
+  if (prefix_command.empty()) {
     return;
   }
 
   Args prefix;
-  char* e = x_strdup(prefix_command);
-  char* saveptr = nullptr;
-  for (char* tok = strtok_r(e, " ", &saveptr); tok;
-       tok = strtok_r(nullptr, " ", &saveptr)) {
-    char* p;
-
-    p = find_executable(ctx, tok, MYNAME);
-    if (!p) {
-      fatal("%s: %s", tok, strerror(errno));
+  for (const auto& word : Util::split_into_strings(prefix_command, " ")) {
+    std::string path = find_executable(ctx, word.c_str(), MYNAME);
```

Comment:
Example: Can NOT be easily converted to string_view due to lacking null-termination. Changing find_executable is not trivial.